kikar_hamedina/kikar_hamedina/settings.py

```diff
@@ -123,15 +125,18 @@
 
 WSGI_APPLICATION = 'kikar_hamedina.wsgi.application'
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'kikar',
-        'USER': 'kikar',
-        'PASSWORD': 'kikar',
-        'HOST': 'localhost'
-    }
-}
+# DATABASES = {
+#     'default': {
+#         'ENGINE': 'django.db.backends.postgresql_psycopg2',
+#         'NAME': 'kikar',
+#         'USER': 'kikar',
+#         'PASSWORD': 'kikar',
+#         'HOST': 'localhost'
+#     }
+# }
+config = dj_database_url.config(default='postgres://kikar:kikar@localhost:5432/kikar')
```

Comment:
@yotammanor see [here](https://github.com/kennethreitz/dj-database-url) :
urischeme://username:password@uri_to_db:port/db_name

This is a URI standard. and thus a convention used in the almost all services: rabbitmq, mongodb, redis, elasticsearch. etc
web URL which is a specific type of URI also conforms:
for example for a site with basic http auth you can use
http(schema type)://user:password@url_to_website/specific_resource